src/App.tsx

```diff
@@ -1,95 +1,116 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
+import goodsFromServer from './server/goodsFromServer.json';
+import { GoodList } from './components/GoodList';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
-
-enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
-}
-
-type ReorderOptions = {
-  sortType: SortType,
+type State = {
+  sortType: string,
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+export class App extends React.Component<{}, State> {
+  state = {
+    sortType: 'NONE',
+    isReversed: false,
+  };
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  render() {
+    const { sortType, isReversed } = this.state;
+    const visibleGoods = [...goodsFromServer];
 
-  return visibleGoods;
-}
+    visibleGoods.sort((g1, g2) => {
```

Comment:
it would be better to create a separate function `getReorderedGoods` outside the class for this